mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -30,27 +31,34 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 	FIELD f_eyfbjeig horizontalSpeed F
 	FIELD f_eyihyrff pistonMovementDelta [D
 	FIELD f_fbrwyrto inNetherPortal Z
+	FIELD f_fhrqlvrj DELTA_AFFECTED_BLOCKS_BELOW D
```

Comment:
I think yarn's name for this is better while also fits with what we already have (`getVelocityAffectingPos`, where it's used)
```suggestion
	FIELD f_fhrqlvrj VELOCITY_AFFECTING_POS_Y_OFFSET D
```